backend/tests/reservation/getAllForRestaurant.js

```diff
@@ -0,0 +1,264 @@
+import chai, { assert } from 'chai';
+import chaiHttp from 'chai-http';
+import config from '../../config';
+import connection from '../../database';
+
+chai.use(chaiHttp);
+// Testing /reservation/?restaurantID GET endpoint. Using IDs 1-5
+
+before(async () => {
+  // Setup the databse before any tests in this file run.
+  const errors = [];
+  await connection
+    .asyncQuery('INSERT INTO restaurant_db.RESTAURANT VALUES (1, "Example resteraunt");')
```

Comment:
nit pick, but **example restaurant**? or is it supposed to be spelt different as an example of a named restaurant? unsure